test/kafka-sender-spec.ts

```diff
@@ -147,330 +122,27 @@ describe('Kafka Sender', () => {
         }
     });
 
-    it('should call flush once per run and before the buffer is full', () => {
```

Comment:
This file as so many tests removed

Reply:
im not fully sure what you mean here, most of these test were just moved to another file.